### ISSUE TITLE
Different Language Binding

### ISSUE THREAD

**Issue by sirinath** [libdynd/libdynd#689]
Can you consider different language bindings along with python version?

**Comment by mwiebe** [comment]
Yes, supporting more language bindings is a very important part of where we want to take DyND. Currently, we're focused on the C++ and Python interfaces, working to get both of them functioning very well before beginning any other bindings. If anyone is interested in giving additional bindings a try, we're definitely interested and would support that effort. Which languages interest you most?

**Comment by sirinath** [comment]
I am interested JVM and .Net support. Maybe Julia support might be important.

**Comment by testmana2** [comment]
Julia language +1

**Comment by izaid** [comment]
@testmana2 Can you elaborate a little on why you might want Julia bindings? Not that it's a bad idea, I'm just curious...
